internal/commands/build/build.go

```diff
@@ -128,13 +123,8 @@ func runBuild(dockerCli command.Cli, application string, opt buildOptions) (refe
 		return nil, err
 	}
 
-	if ref == nil {
-		if ref, err = computeDigest(bundle); err != nil {
-			return nil, err
-		}
-	}
-
-	if err = packager.PersistInBundleStore(ref, bundle); err != nil {
+	ref, err = packager.PersistInBundleStore(ref, bundle)
```

Comment:
You can just return this

Reply:
right. On initial build PR there was a few subsequent function calls before return